### ISSUE TITLE
man page for alr

### ISSUE THREAD

**Issue by stcarrez** [alire-project/alire#376]
It would be nice to have a man page for alr command.

Not a man page that redirects to another documentation. A real man page :-)

I can help if you wish. Let me know the basic format (Troff is nice) but I guess you'll prefer RST :-).

**Comment by mosteo** [comment]
Thanks for the kind offer :-) Personally I've never used RST until this project, nor Troff ever. If Troff is the norm for man pages, I'd say why not.

If you're testing `alr` in a linux box, don't miss this one: #357

**Comment by stcarrez** [comment]
Yes, man pages are exclusively in Troff (which I really love...). Tools such as Pandoc or Sphinx can convert RST to Troff, PDF and other formats...

**Comment by Fabien-Chouteau** [comment]
If we do a man page I think it should be automatically extracted from the help text in the tool itself.

**Comment by stcarrez** [comment]
Hum... Yes and no. The help information as available today is too short for a man page.
A real man page, I mean a man page that is useful, needs more details.

Now, it may be easier to generate the tool built-in help from documentation files.

Apart from that, on Unix, it could be interesting to have a git-like implementation for the help
where we run the man command on the given alire command. For Windows, there could be some
pre-formatted files to display so the man command could be emulated...
